lib/perl/Genome/Model/Tools/SpeedSeq/Align.pm

```diff
@@ -0,0 +1,78 @@
+package Genome::Model::Tools::SpeedSeq::Align;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::SpeedSeq::Align {
+    is => 'Genome::Model::Tools::SpeedSeq::AlignBase',
+    has_input => [
+        fastq => {
+            is => 'Text',
+            doc => 'paired-end fastq file. if -p flag is used then expected to be an interleaved paired-end fastq file, and fastq2 may be omitted. (can be gzipped)',
```

Comment:
It might be better if the doc refers to the `--paired` option of this tool instead of `-p`.